src/footer/bulib-footer.min.js

```diff
@@ -0,0 +1,8 @@
+import BULibHours from '../libhours/libhours';
+window.customElements.define("bulib-hours", BULibHours);
```

Comment:
we're not actually using this file, and if we were, we would want to leave this import to `locoso` anyway.